frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.tsx

```diff
@@ -20,32 +24,55 @@ export const SharingPaneButton = ({
   title,
   onClick,
   badge,
-  externalLink = false,
   "data-testid": dataTestId,
-}: SharingOptionProps) => (
-  <Paper
-    className={S.Container}
-    p={24}
-    pt={52}
-    withBorder
-    data-testid={dataTestId}
-    onClick={onClick}
-    mih="100%"
-    pos="relative"
-    w={"22rem"}
-  >
-    <Stack>
-      {externalLink && (
-        <Box pos="absolute" top={12} right={12}>
-          <Icon name="share" color="var(--external-link-icon-color)" />
+  isDisabled,
+  disabledLink,
+}: SharingOptionProps) => {
+  return (
+    <Paper
+      component="article"
+      className={cx(S.Container, { [S.Disabled]: isDisabled })}
+      p={24}
```

Comment:
I know it's not your code but could we change these to a string value with `24px` or `1.5rem` (rems preferred for me)? Right now this value and the `pt` strike me as magic numbers

Reply:
I like the text value with unit since it seems to be much more clear what it's supposed to mean.

Reply:
Fixed in 004d246310f7ee1f924d2b0808400dda12d2f8c5